cogs/torrents.py

```diff
@@ -0,0 +1,70 @@
+import os
+import discord
+import requests
+import json
+from discord.ext import commands
+
+
+def load_json(token):
+    with open('./config.json') as f:
+        config = json.load(f)
+    return config.get(token)
+
+
+def setup(client):
+    client.add_cog(Torrents(client))
+
+
+def get_magnet(search: str) -> discord.Embed:
+    """
+    Get a magnet link for a torrent.
+    Uses the backend api for 'the pirate bay' to find torrent hash and name.
+    Create the magnet link based on the information returned.
+    Present to the user with helpful information.
+    """
```

Comment:
I think something better is:
Get a magnet link for a torrent.
Usage: !torrent <search>

Something like that. It doesn't need all the technical details. This is what shows up when the user types !help torrent

Reply:
Ahhhhh ok makes sense.